File: datastructure/Stack_Queue/Stack.js
// Stack
class Stack{
    constructor(){
        this.value = [];
    }

    push(value){
        this.value.push(value);
    }

    pop(){
        return this.value.pop();
    }

    is_empty(){
        return this.value.length === 0;
    }
}

// Example
let my_stack = new Stack();
my_stack.push(12);
my_stack.push(24);
console.log(my_stack.is_empty())

File: datastructure/Stack_Queue/Stack.py
# Stack
class Stack:
    def __init__(self) -> None:
        self.value = []
        
    def push(self, value):
        self.value.append(value)
        
    def pop(self):
        return self.value.pop()
    
    def is_empty(self) -> bool:
        return len(self.value) == 0
    

# Example
my_stack = Stack()
my_stack.push(12)
my_stack.push(24)
print(my_stack.is_empty())